### PR TITLE
Blog landing page

### PR DESCRIPTION
**Pull request by torkjels**
Statically generating the pagination will be done in a separate PR.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit bc9d053f9789b73a72477c4b969a0f517e5e8b41

https://deploy-preview-169--masifunde.netlify.com

File: api/blog.js [hunk 1]
```diff
@@ -57,3 +58,61 @@ export async function fetchBlogPostPage(locale, slug) {
...
+        totalNumberOfPages,
+        isLastPage,
+        blogPosts,
+      })
+    })
+}
+
+export async function fetchBlogLandingPage(locale, page = 1) {
+  const pageContent = await fetchSingleEntry('pageBlogHome', locale)
+  const fetchBlogPostsResult = await fetchBlogPostsList(locale, page)
```
Comment (denisw) [inline api/blog.js]: You can still `await Promise.all([...])` to make the two requests run in parallel.

**Comment by torkjels** [comment]
@denisw I've taken care of what you mentioned! I didn't add any comments to the BlogListNavButton logic, but maybe it's still needed even after the refactor?

**Comment by denisw** [comment]
Looks good to me! I think after the last fixes we talked about it's ready for merging. 🚀

**Comment by denisw** [comment]
@torkjels The only issue needing fixing is the shrinking Next button on IE 11.

**Comment by torkjels** [comment]
@denisw Fertig!

**Comment by denisw** [comment]
@torkjels Perfekt!

**Pull request merged**